lib/rmt/mirror.rb

```diff
@@ -109,6 +111,11 @@ def mirror_metadata(repository_dir, repository_url, temp_metadata_dir)
       if (e.http_code == 404)
         logger.info(_('Repository metadata signatures are missing'))
       else
+        if n_retry > 0
+          sleep(2)
```

Comment:
I suggest to add a log line here including the http error, so it's visible when a retry happens.

Reply:
Done